src/main/java/com/vasivkov/start/domain/Reserve.java

```diff
@@ -0,0 +1,30 @@
+package com.vasivkov.start.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "reserves")
+public class Reserve {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
```

Comment:
As there are some annotations better to have empty lines between fields to increase readability.